pkg/blockerslack/reporters/blockers/blockers_reporter.go

```diff
@@ -36,6 +36,14 @@ const (
 	//upcomingSprintMsgFmt = "There are %s which do not have _UpcomingSprint_.\nPlease apply this keyword if the bug will not be resolved during this sprint\n"
 )
 
+var (
+	seriousKeywords = []string{
```

Comment:
I'm trying to decide if we should put this definition in pkg/bugs for some later user. But I guess this is fine.

Reply:
I am fine either way, feel free to adjust without checking in with me 😉